templates/pages/object.html

```diff
@@ -29,6 +29,7 @@
 
         {{#if related.people}}{{> records/record-related-people }}{{/if}}
         {{#if related.organisations}}{{> records/record-related-organisations }}{{/if}}
+        <div id="articles"></div>
 
         {{!-- {{> records/panel-taxonomy}} moved lower to record-details. but panels could also be used here. --}}
```

Comment:
Doesn't belong to this PR but maybe we can delete this commented line in the future?